inst/testthat/helper_misc.R

```diff
@@ -17,6 +17,14 @@ with_future = function(backend, expr, ...) {
   force(expr)
 }
 
+with_mirai = function(expr) {
+  requireNamespace("mirai")
+  mirai::daemons(1)
+  on.exit(mirai::daemons(0), add = TRUE)
```

Comment:
will this somehow influence mirai sessions that the user has open for another reason?

Reply:
Maybe we can use an `.mlr3` compute profile to avoid this

Reply:
sry I see this is only for tests